tiledb/ml/_cloud_utils.py

```diff
@@ -26,3 +34,18 @@ def get_s3_prefix(namespace: str) -> Union[str, None]:
             return os.path.join(organization.default_s3_path, CLOUD_MODELS)
         else:
             return None
+
+
+def update_file_properties_wrapper(
```

Comment:
Because of the fact that each time we want to update a file property we have to update all of them (this should be fixed sometime in the future), I would pass uri and a dictionary of the file properties as arguments. We will then handle values etc during the call.

Reply:
Okay will fix it!

Reply:
i'd remove the `_wrapper` from the name